bridge_test.go

```diff
@@ -51,6 +53,47 @@ func TestHandlePacket(t *testing.T) {
 				&TimerEvent{
 					metricName: "foo",
 					value:      200,
+					labels:     map[string]string{},
+				},
+			},
+		}, {
+			name: "datadog tag extension",
+			in:   "foo:100|c|#tag1:bar,tag2:baz,tag3,tag4",
+			out: Events{
+				&CounterEvent{
+					metricName: "foo",
+					value:      100,
+					labels:     map[string]string{"tag1": "bar", "tag2": "baz", "tag3": ".", "tag4": "."},
+				},
+			},
+		}, {
+			name: "datadog tag extension with # in all keys (as sent by datadog php client)",
+			in:   "foo:100|c|#tag1:bar,#tag2:baz,#tag3,#tag4",
+			out: Events{
+				&CounterEvent{
+					metricName: "foo",
+					value:      100,
+					labels:     map[string]string{"tag1": "bar", "tag2": "baz", "tag3": ".", "tag4": "."},
```

Comment:
`<n/a>` or dot: Wouldn't `<n/a>` make querying such metrics a bit awkward? Like this `http_requests_total{live="<n/a>"}`

Besides, I'm yet to find a single useful example of a valueless tag in datadog docs, maybe we shouldn't implement this at all? Ok, now it sounds like bikeshedding. I'll change it.